index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js" defer></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="homepage">
+        <nav id="navbar">
```

Comment:
- [ ] I see that you have used `<nav>` tag which means you understand its usage 😄 . Well, I think it will be better to remove the logo inside the nav as the nav should only contain your navigation elements which you will be adding in your subsequent milestone this way, each part of your code will have its own distinct meaning and usage hence improves accessibility 👍🏽 

e,g 

```
<header>
   <a href="" >logo </a>
    <nav>
         ...hamburger (future navigation elements)
     </nav>
    </header>
```